packages/synapse-react-client/package.json

```diff
@@ -162,7 +162,7 @@
     "@vitejs/plugin-legacy": "^3.0.2",
     "@vitejs/plugin-react": "^3.0.1",
     "@welldone-software/why-did-you-render": "^7.0.1",
-    "babel-jest": "^28.1.1",
+    "babel-jest": "^29.4.3",
```

Comment:
Was running into what I think was an incompatibility between Jest 29.x and babel-jest 28.x that manifested as a stack overflow error. Issue went away when I aligned them